.github/workflows/gitlab-ci.yaml

```
@@ -2,95 +1,81 @@
-  services:
-    - docker:dind
-  script:
-    - pip install pip-tools
-    - pip-compile --output-file requirements.tmp.txt requirements.in
-    - cat requirements.tmp.txt
-    - echo "take Diff:"
-    - diff --ignore-matching-lines=pip-compile --ignore-all-space requirements.txt requirements.tmp.txt
-  allow_failure: true
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)